Buildings/Controls/OBC/ASHRAE/G36/AHUs/MultiZone/VAV/Validation/Controller.mo

```diff
@@ -120,15 +120,12 @@ model Controller "Validation controller model"
     final realFalse=0)
     "Convert to real"
     annotation (Placement(transformation(extent={{190,70},{170,90}})));
-  Buildings.Fluid.BaseClasses.ActuatorFilter fil(
-    final f=5/(2*Modelica.Constants.pi*10),
-    initType=Modelica.Blocks.Types.Init.InitialOutput,
-    final y_start=0)
-    "Filter signal"
-    annotation (Placement(transformation(extent={{160,70},{140,90}})));
   Buildings.Controls.OBC.CDL.Reals.GreaterThreshold greThr(final t=0.5)
     "Compare filtered signal to threshold to trigger true status"
     annotation (Placement(transformation(extent={{130,70},{110,90}})));
+  Buildings.Controls.OBC.CDL.Discrete.ZeroOrderHold sam(samplePeriod=10)
```

Comment:
@AntoineGautier 
This change is not good. It is implemented as
![image](https://github.com/lbl-srg/modelica-buildings/assets/1329080/4f04ab26-2ece-40b6-89f8-e7b4b70c6de5)
with a 10 second sampling time. If this were used in a model with an HVAC system, simulation would be very slow due to the frequent sampling (and users are likely to copy paste such fragments from examples).

Can we use the same patterns as we used last week for the enable signal of the HP templates? Or just use a direct feedthrough. Even a 2nd order filter would be better here I think as the 10 second sampling is very frequent and may also give issues in cross-tool verification.

Reply:
This was refactored in fc5e57b9cf using a timer.
As explained in https://github.com/lbl-srg/modelica-buildings/issues/3923#issuecomment-2202418450 (which I updated with the alternative implementation using a timer), the implementation using a timer does not systematically break algebraic loops.
However, in the validation models in which it is used here, it accomplishes this goal without using classes outside the CDL package, as Jianjun suggested.